crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -618,25 +622,26 @@ pub fn view_3d(
             state,
             hovered_context,
             egui::Color32::WHITE,
-        );
+        )?;
     }
 
-    {
-        let mut box_batch = line_builder.batch("scene_bbox");
-        if state.state_3d.show_bbox {
-            box_batch
-                .add_box_outline(&state.bounding_boxes.current)
-                .map(|lines| lines.radius(Size::AUTO).color(egui::Color32::WHITE));
-        }
-        if state.state_3d.show_accumulated_bbox {
-            box_batch
-                .add_box_outline(&state.bounding_boxes.accumulated)
-                .map(|lines| {
-                    lines
-                        .radius(Size::AUTO)
-                        .color(egui::Color32::from_gray(170))
-                });
-        }
+    if state.state_3d.show_bbox {
+        line_builder
+            .reserve(1, 5)? // TODO(andreas): hardcoded knowledge of box->lines is confusing.
```

Comment:
To me this reads like "one strip of 5 vertices" , which is really weird too. Why 5? There should be 8 vertices on a 3D box, and 12 edges.